app/views/candidate_mailer/application_rejected_all_rejected.erb

```diff
@@ -1,3 +1,37 @@
+<% if FeatureFlag.active?('apply_again_email_content') %>
+
+Dear <%= @application_form.first_name %>,
+
+# Application decision
+
+<%= @course.provider.name %> has decided not to progress your teacher training application for <%= @course.name_and_code %> on this occasion.
+
+# You may still be able to get a place on a course
+
+Your <%= @application_form.application_choices.count > 1 ? 'provider has' : 'providers have' %> left you some feedback. There may be things you can do to get a place on a course this year, next year or in the future.
+
+Review your feedback:
+
+https://www.apply-for-teacher-training.education.gov.uk/candidate/account
```

Comment:
Are we no longer showing the feedback in our rejection email? cc @EmmaFrith @paulrobertlloyd

Reply:
This is looking awesome thanks @davidgisbey!

@fofr @paulrobertlloyd - I took the view to not include the reasons for rejection in this final email because a) the main thing we want them to do is to log back in and apply again b) we want them to review all their feedback as a whole to see if they're suitable for applying again c) the email is already long and we don't want them to miss the 'apply again' call to action. Does this sound sensible?